docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/import-eks-cluster-enable-hybrid-mode.md

```diff
@@ -0,0 +1,517 @@
+    | Root CA Certificate | The PEM-encoded certificate of your Certificate Authority (CA) that serves as the trust anchor. This certificate is used by IAM Roles Anywhere to validate the authenticity of the client certificates presented by your hybrid nodes. |                                                                                                  |
+    | Root CA Private Key | The private key corresponding to your CA's certificate, used to sign client certificates.                                                                                                                                              |                                                                                                  |
+
+14. Click **Save Changes** when complete.
+
+15. If you are using IAM Roles Anywhere, check whether the `aws-auth` ConfigMap exists.
+
+    ```shell
+    kubectl get configmap aws-auth --namespace kube-system
+    ```
+
+    Output if the ConfigMap is found.
+
+    ```shell
+    NAME       DATA   AGE
+    aws-auth   1      15d
+    ```
+
+    If it does not exist, you will receive an error similar to
+    `Error from server (NotFound): configmaps "aws-auth" not found`.
+
+16. If the `aws-auth` ConfigMap does not exist, create the following ConfigMap in the `kube-system` namespace using the
+    following command.
+
+    Ensure to replace `<roleArn>` with the **Role ARN** entry from step 13.
+
+    ```shell
+    kubectl create -f=/dev/stdin <<-EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: aws-auth
+      namespace: kube-system
+    data:
+      mapRoles: |
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: <roleArn>
+          username: system:node:{{SessionName}}
+    EOF
+    ```
+
+17. If the `aws-auth` ConfigMap already exists, append the `mapRoles` entry in your existing ConfigMap.
+
+    You can edit the existing configmap using the following command.
+
+    ```shell
+    kubectl edit configmap aws-auth --namespace kube-system
+    ```
+
+    The following example shows the `mapRoles` entry appended below an existing entry. Ensure to replace `<roleArn>`
+    with the **Role ARN** entry from step 13.
+
+    ```yaml {13-17} hideClipboard
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: aws-auth
+      namespace: kube-system
+    data:
+      mapRoles: |
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: arn:aws:iam::111122223333:role/eks-nodegroup-example
+          username: system:node:{{EC2PrivateDNSName}}
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: <roleArn>
+          username: system:node:{{SessionName}}
+    ```
+
+18. If you need to configure proxies for edge host communication or you plan to use VPN connectivity for your edge
```

Comment:
"you plan to use VPN connectivity for your edge hosts" -> "you plan to specify a VPN server IP for one or more edge hosts"